src/auth.ts

```diff
@@ -18,7 +18,9 @@ async function signOut() {
     name: WORKOS_COOKIE_NAME || 'wos-session',
   };
   if (WORKOS_COOKIE_DOMAIN) cookie.domain = WORKOS_COOKIE_DOMAIN;
-  cookies().delete(cookie);
+
+  const nextCookies = await cookies();
+  nextCookies.delete(cookie);
```

Comment:
Confirming this also works for Next 14?

Reply:
Tested that it works on the next-authkit-example app, which is still on Next 14.